templates/core/home.html

```diff
@@ -81,16 +81,6 @@ <h1 class="brand">{{ "cubes" | fas(fw=true) }} Docs.rs</h1>
             return String.fromCharCode(ev.charCode || ev.keyCode);
         }
 
-        document.getElementById("i-am-feeling-lucky-button").onclick = function () {
-            var form = document.getElementsByClassName("landing-search-form");
-            var input = document.createElement('input');
-            input.type = 'hidden';
-            input.name = 'i-am-feeling-lucky';
-            input.value = 1;
-            document.getElementsByClassName("landing-search-form")[0].appendChild(input);
-            return true;
-        };
```

Comment:
Are you sure just adding the `name` and `value` has the same behavior? That doesn't create the form or hide the element. Do you know what the form was for?

Reply:
It's not creating a form. It's adding a new `input` element to the existing form.

Yes, adding the name does the same thing. The resulting URL, `/search?query=regex&i-am-feeling-lucky=1`, is the same before and after this change.